Library/Homebrew/formula_installer.rb

```diff
@@ -219,6 +219,10 @@ def build_bottle_postinstall
   end
 
   def install
+    if DevelopmentTools.installed? && !pour_bottle?
```

Comment:
any appreciable difference in speed depending on the order of those conditions?

Reply:
Yeh, the latter is likely to be a bit faster so probably preferable?